### PR TITLE
Changed the shape of speaker icons in the timeline grid from rounded rectangle to circles

### PR DESCRIPTION
**Pull request by SEKI-YUTA**
## Issue
- close https://github.com/DroidKaigi/conference-app-2024/issues/224

## Overview (Required)
- Changed the shape of speaker icons in the timeline grid from rounded rectangle to circles.

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

## Movie (Optional)
Before | After
:--: | :--:
<video src="" width="300" > | <video src="" width="300" >

### CODE REVIEW

File: feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableGridItem.kt [hunk 1]
```diff
@@ -240,7 +241,7 @@ private fun SpeakerIcon(
         contentDescription = stringResource(SessionsRes.string.content_description_user_icon),
         modifier = modifier
             .size(TimetableGridItemSizes.speakerHeight)
-            .clip(RoundedCornerShape(8.dp))
+            .clip(CircleShape)
             .border(
                 BorderStroke(1.dp, MaterialTheme.colorScheme.outline),
                 RoundedCornerShape(8.dp),
```
Comment (takahirom) [inline feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableGridItem.kt:247]: Thanks! Could we change as well to fix the border?

Comment (SEKI-YUTA) [reply feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableGridItem.kt:247]: Sorry. I missed this point. I will fix this point.

Comment (SEKI-YUTA) [reply feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableGridItem.kt]: I fixed this point.

**Review by takahirom — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**